x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/dashboards/dashboards_card_config.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import {
+  CardSelectorListItemAssetType,
+  type CardSelectorListItem,
+} from '../common/card_selector_list';
+import { DashboardsCardItemId } from './types';
+
+const VIDEO_SOURCE = '//play.vidyard.com/K6kKDBbP9SpXife9s2tHNP.html?autoplay=1';
+
+export const DASHBOARDS_CARD_ITEMS: CardSelectorListItem[] = [
```

Comment:
We have an existing `dashboardCardConfig` in `x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/dashboards/index.ts`, probably we can put `DASHBOARDS_CARD_ITEMS` into the existing index.ts file and as part of its property?

In this case might be able to know which cards needed to be `delayed when closing` and `useDelayedVisibility` in `x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/onboarding_card_panel.tsx`?

Then we might be able to remove `isExpanded={expandedCardId === id}` added to `LazyCardComponent` in `x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/onboarding_body.tsx`? and avoid controlling the closing status in each card?

Reply:
You are right! I've already added the`useDelayedVisibility` in a higher level and I removed the `isExpanded={expandedCardId === id}` from the `LazyCardComponent`